src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -789,14 +790,14 @@ private function fixPrevName(Tokens $tokens, int $index, array $uses, string $na
                 break;
             }
 
-            if ($tokens[$index]->equalsAny([[T_STRING], [T_NS_SEPARATOR]])) {
+            if ($tokens[$index]->isGivenKind([T_STRING, T_NS_SEPARATOR])) {
                 $typeStartIndex = $index;
                 if (null === $typeEndIndex) {
                     $typeEndIndex = $index;
                 }
             } else {
                 if (null !== $typeEndIndex) {
-                    $index += $this->shortenClassIfPossible($tokens, $typeStartIndex, $typeEndIndex, $uses, $namespaceName);
+                    $this->shortenClassIfPossible($tokens, $typeStartIndex, $typeEndIndex, $uses, $namespaceName);
```

Comment:
why not fast-forward the index anymore ?

Reply:
It is not used here. After the `if` the `break` finishes the `while` loop and the function ends.

Reply:
makes sense 👍🏻 
(pssst, consider to have dedicated commit with commit-log describing that, you would avoid such question during review ;) )